tests/scripts/import_book.py

```diff
@@ -68,35 +77,35 @@ def do_create():
     dataset = tablib.Dataset(*rows, headers=['id', 'name', 'author_email', 'price'])
 
     book_resource = _BookResource()
+    print(f"before create: {Book.objects.count()} Books")
     do_import(book_resource, dataset)
-
-    assert Book.objects.count() == NUM_ROWS * 2
+    print(f"after create: {Book.objects.count()} Books")
     Book.objects.all().delete()
 
 
 def do_update():
     rows = [('', 'Some new book', 'email@example.com', '10.25')] * NUM_ROWS
     books = [Book(name=r[1], author_email=r[2], price=r[3]) for r in rows]
-    Book.objects.bulk_create(books)
-    assert NUM_ROWS == Book.objects.count()
 
-    # deletes - there must be existing rows in the DB...
-    # i.e. so they can be deleted
+    # run 'update' - there must be existing rows in the DB...
+    # i.e. so they can be created
+    #Book.objects.bulk_create(books)
+
+    # find the ids, so that we can perform the update
     all_books = Book.objects.all()
     rows = [(b.id, b.name, b.author_email, b.price) for b in all_books]
-    # Add this line in order to perform bulk delete
     dataset = tablib.Dataset(*rows, headers=['id', 'name', 'author_email', 'price'])
 
     book_resource = _BookResource()
+    print(f"before update: {Book.objects.count()} Books")
     do_import(book_resource, dataset)
+    print(f"after update: {Book.objects.count()} Books")
 
-    assert Book.objects.count() == NUM_ROWS
-    Book.objects.all().delete()
```

Comment:
Why are the asserts removed? I don't see them moved anywhere else.

I understand you may want to remove this and the `delete()` call for profiling reasons, but somewhat we should be testing that the results are correct, maybe pass a flag and run the test again?

Reply:
I've re-added the asserts and also some further enhancements to the script.